src/view/com/util/LoadingPlaceholder.tsx

```diff
@@ -267,6 +271,18 @@ const styles = StyleSheet.create({
     paddingHorizontal: 10,
     paddingTop: 20,
     paddingBottom: 5,
+    paddingRight: 15,
+  },
+  postCtrls: {
+    opacity: 0.5,
```

Comment:
This is better than using different stroke width because it keeps spacing intact.